src/utils/tests/text.spec.js

```diff
@@ -0,0 +1,12 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+import { truncateText } from "../text";
+
+describe("text", () => {
+  it("should truncate the text", () => {
+    const text = "this is a very long text and ends here";
+    expect(truncateText(text, 30)).toMatch("this is a ver...and ends here");
+  });
+});
```

Comment:
I know this seems weird but a test with a short string confirming it's unchanged would be good. :)

Reply:
Done.